### PR TITLE
Form Recognizer consistency updates part 2

### PR DESCRIPTION
**Pull request by samvaity**

### CODE REVIEW

**Comment by mssfang** [comment]
missing motes on changelog?

File: sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/administration/DocumentModelAdministrationAsyncClient.java [hunk 1]
```diff
@@ -824,32 +824,32 @@ public Mono<ModelOperation> getOperation(String operationId) {
...
+     *     if &#40;ModelOperationStatus.FAILED.equals&#40;modelOperationInfo.getStatus&#40;&#41;&#41;&#41; &#123;
+     *         System.out.printf&#40;&quot;Operation fail error: %s%n&quot;, modelOperationInfo.getError&#40;&#41;.getMessage&#40;&#41;&#41;;
      *     &#125;
      * &#125;&#41;;
      * </pre>
      * <!-- end com.azure.ai.formrecognizer.administration.DocumentModelAdministrationAsyncClient.getOperationWithResponse#string -->
      *
      * @param operationId Unique operation ID.
-     * @return A {@link Response} containing the requested {@link ModelOperation}.
+     * @return A {@link Response} containing the requested {@link DocumentModelOperationInfo}.
```
Comment (catalinaperalta) [inline sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/administration/DocumentModelAdministrationAsyncClient.java:840]: From our discussion with Paul it wasnt clear to me that ModelOperation/ModelOperationInfo could follow the same pattern as DocumentModelInfo/DocumentModelSummary. Did Paul agree to that in the end?

Comment (samvaity) [reply sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/administration/DocumentModelAdministrationAsyncClient.java:840]: I thought we wanted to keep this naming regardless of if we could get that model pattern in the swagger.

@witemple-msft @HarshaNalluru @kinelski

**Comment by samvaity** [comment]
Handling in #29978

**Pull request closed without merging**